### PR TITLE
Migrate to the latest badge standard

### PR DESCRIPTION
**Pull request by DJMcNab**
Also clean up the intra-doc links to not use `concat`

I'm planning on documenting this pattern for intra-doc links with a doc include on https://linebender.org, so that we have something to point towards. For this, I thought I'd better check that the pattern is as clean as possible, and it turns out that the `concat` is already implied, so we can just use that.

### CODE REVIEW

File: tracing_android_trace/README.md [hunk 1]
```diff
@@ -32,7 +31,7 @@ This limitation is due to the underlying Android platform APIs.
 <img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread.">
 <figcaption>
 
-Tracing spans for [Vello](https://github.com/linebender/vello) shown in Android GPU Inspector
+Tracing spans for [vello](https://github.com/linebender/vello) shown in Android GPU Inspector
```
Comment (xStrom) [inline tracing_android_trace/README.md:34]: If the idea behind lower case is to reference the crate then let's tick it.

```suggestion
Tracing spans for [`vello`](https://github.com/linebender/vello) shown in Android GPU Inspector
```

File: android_trace/README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
-<div align="center">
+<div align="center" class="rustdoc-hidden">
 
 # Android Trace
 
 </div>
 <!-- Close the <div> opened in lib.rs for rustdoc, which hides the above title -->
```
Comment (xStrom) [inline android_trace/README.md:6]: You removed the comment from the other readme, so I guess it should be removed here too as the closing `div` was removed.

```suggestion
```

**Review by xStrom — APPROVED** [review]
Looks good. Verified both with `cargo doc` locally too.

The license section is hidden, so I guess the links are defined properly just in case. Making note of it here, in case you intetended to unhide that section.

File: tracing_android_trace/README.md (hunk 1 above)
Comment (DJMcNab) [reply tracing_android_trace/README.md:34]:
```suggestion
Tracing spans for [Vello](https://github.com/linebender/vello) shown in Android GPU Inspector
```
This change was a copy-paste mistake; I wasn't intending to change this line.
Thanks for catching.

**Comment by DJMcNab** [comment]
> The license section is hidden, so I guess the links are defined properly just in case. Making note of it here, in case you intended to unhide that section.

Indeed, I wasn't intending to unhide the section. This was done for a few reasons:
- To make sure that the link is valid in the HTML, even though it's unreachable except with view source. There's no real cost to it, and it's good practise in case the hiding stops working in future, or we decide to stop hiding those sections
- More importantly, to have it available as an example for https://github.com/linebender/linebender.github.io/pull/55 :)

I'll followup with a changelog and 0.1.1 release.

**Pull request merged**